py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -3435,3 +3435,24 @@ def aten_ops_prelu(
         args[0],
         args[1],
     )
+
+
+@dynamo_tensorrt_converter(
+    torch.ops.aten.arange.start_step, supports_dynamic_shapes=True
+)
+def aten_ops_arange_start_step(
+    ctx: ConversionContext,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return impl.arange.arange(
+        ctx,
+        target,
+        SourceIR.ATEN,
+        name,
+        start=args[0],
+        end=args[1],
+        step=args_bounds_check(args, 2, 1),
+    )
```

Comment:
yes, arange is already implemented in op_evaluatiors.py, but the definition and implementation is in the same place, I have moved the implementation to a seperate impl.arange.arange function, so that it is easier for other operators to reuse this arange.